h2o-core/src/main/java/water/api/schemas3/ModelOutputSchemaV3.java

```diff
@@ -61,6 +61,9 @@
   @API(help="Cross-validation model metrics summary", direction=API.Direction.OUTPUT, level=API.Level.critical)
   public TwoDimTableV3 cross_validation_metrics_summary;
 
+  @API(help="TargetEncoderModel key", direction=API.Direction.OUTPUT, level=API.Level.critical)
+  public KeyV3.ModelKeyV3 te_model;
```

Comment:
@sebhrusen I assume that for output's fields we are ok without `internal` prefix